SPECS/kernel/config

```diff
@@ -97,8 +99,10 @@ CONFIG_POSIX_CPU_TIMERS_TASK_WORK=y
 CONFIG_TICK_ONESHOT=y
 CONFIG_NO_HZ_COMMON=y
 # CONFIG_HZ_PERIODIC is not set
-CONFIG_NO_HZ_IDLE=y
-# CONFIG_NO_HZ_FULL is not set
+# CONFIG_NO_HZ_IDLE is not set
+CONFIG_NO_HZ_FULL=y
+CONFIG_CONTEXT_TRACKING=y
+# CONFIG_CONTEXT_TRACKING_FORCE is not set
```

Comment:
Context tracing allows traces for when no hz full is on https://github.com/torvalds/linux/commit/24a9c54182b3758801b8ca6c8c237cc2ff654732

https://unix.stackexchange.com/questions/579918/how-do-i-switch-on-config-context-tracking-in-linux